apps/lib/signer/signer.go

```diff
@@ -20,6 +22,19 @@ import (
 	"github.com/rs/zerolog"
 )
 
+// gmorkworld in ascii
+const StorkMagicNumber = "103109111114107119111114108100"
```

Comment:
first pass at a magic number - converted "gmorkworld" to ascii and concatted together all the values.